src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+const BASE_URL = 'https://mate.academy/students-api';
+
+export const loadUserComments = async (postId: number) => {
```

Comment:
Comments are related to a particular post, but not to a user
```suggestion
export const loadPostComments = async (postId: number) => {
```